### PR TITLE
Add saravgit redlands recommendation File

### LINKED ISSUE #20: Create a file for Redlands Recommendations
# Redlands

- [ ] Create a branch
- [ ] Add a commit
- [ ] Open a Pull Request
- [ ] Have a Discussion
- [ ] Seek a approval
- [ ] Merge Pull Request

### PR DESCRIPTION
**Pull request by saravgit**
closes #20

### CODE REVIEW

File: saravgit-redlands.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Redlands
+
+### Places to Eat
+- Moo
+- Gorment
+- Habbit
+
+### Places to Visit
+- Market Night
```
Comment (githubteacher) [inline saravgit-redlands.md]: Is market night tonight?

**Comment by githubteacher** [comment]
:octocat:

File: saravgit-redlands.md (hunk 1 above)
Comment (saravgit) [reply saravgit-redlands.md]: Its every Thursday

**Review by githubteacher — APPROVED** [review]
Good job! Nice work! 👍 😄 ✨ 🎉

**Pull request merged**